src/quo2/components/wallet/network_routing/view.cljs

```diff
@@ -0,0 +1,197 @@
+                                    :bars-widths-negative bars-widths-negative})
+                                  (animation/move-pressed-bar
+                                   {:bar                  bar
+                                    :bars-widths-negative bars-widths-negative
+                                    :number-previous-bars number-previous-bars})
+                                  (animation/move-next-bars
+                                   {:bars                 next-bars
+                                    :bars-widths-negative bars-widths-negative
+                                    :number-previous-bars (inc number-previous-bars)
+                                    :extra-offset         (- bar-max-width bar-width)
+                                    :add-new-timeout      add-new-timeout-fn}))
+                                (animation/show-max-limit-bar bar-opacity-shared-value)
+                                (reset! selecting-network? true)
+                                (reset! selected-network-idx bar-idx))
+              :on-new-amount (fn [new-amount]
+                               (animation/hide-max-limit-bar bar-opacity-shared-value)
+                               (when on-amount-selected
+                                 (on-amount-selected new-amount @selected-network-idx)))}]))
+
+         (let [{:keys [max-amount network-name]} (some->> @selected-network-idx
+                                                          (nth network-bars))
+               limit-bar-width                   (amount->width max-amount)]
+           [reanimated/view
+            {:style (style/max-limit-bar
+                     {:opacity-shared-value bar-opacity-shared-value
+                      :network-name         network-name
+                      :width                limit-bar-width})}])]))))
+
+
+
+(defn view-internal
+  [{:keys [networks container-style theme] :as params}]
+  (reagent/with-let [timeouts        (atom {})
+                     set-timeout     (fn [k f ms]
+                                       (js/setTimeout (fn []
+                                                        (f)
+                                                        (swap! timeouts dissoc k))
+                                                      ms))
+                     add-new-timeout (fn [k f ms]
+                                       (when-let [existing-timeout (k @timeouts)]
+                                         (js/clearTimeout existing-timeout))
+                                       (swap! timeouts assoc k (set-timeout k f ms)))
+                     total-width     (reagent/atom nil)]
+    [rn/view
+     {:accessibility-label :network-routing
+      :style               (style/container container-style theme)
+      :on-layout           #(reset! total-width (oops/oget % "nativeEvent.layout.width"))}
+     (when @total-width
+       ^{:key (str "network-routing-" (count networks))}
```

Comment:
Why are we using a fixed key based on the count of networks? To play with reconciliation? This doesn't seem reliable, for example, `networks` inner values may change for any reason, but not the count of `networks`.

Reply:
I can't remember why I added this. Maybe a leftover :thinking: I removed and tested and I didn't find any problem.

Also, since the `params` are being passed to the iner component, it'll be rerendered if they change. So I removed it